libraries/AP_Motors/AP_MotorsHeli_Single.cpp

```diff
@@ -512,27 +512,27 @@ void AP_MotorsHeli_Single::servo_test()
 
     if ((_servo_test_cycle_time >= 0.0f && _servo_test_cycle_time < 0.5f)||                                   // Tilt swash back
         (_servo_test_cycle_time >= 6.0f && _servo_test_cycle_time < 6.5f)){
-        _pitch_test += (1.0f / (_loop_rate/2));
-        _oscillate_angle += 8 * M_PI / _loop_rate;
+        _pitch_test += (1.0f / (_loop_rate / 2.0f));
+        _oscillate_angle += 8.0f * M_PI / _loop_rate;
         _yaw_test = 0.5f * sinf(_oscillate_angle);
     } else if ((_servo_test_cycle_time >= 0.5f && _servo_test_cycle_time < 4.5f)||                            // Roll swash around
                (_servo_test_cycle_time >= 6.5f && _servo_test_cycle_time < 10.5f)){
-        _oscillate_angle += M_PI / (2 * _loop_rate);
+        _oscillate_angle += M_PI / (2.0f * _loop_rate);
```

Comment:
Why? This does a float multiplication instead of an integer one without any benefit.